website/docs/components/link/inline/partials/guidelines/guidelines.md

```diff
@@ -14,7 +14,9 @@
 
 The Inline Link is only published as a component in Ember, there is no corresponding component in Figma. To achieve the same results, apply **text** and **color** styles from the [Foundations library](https://www.figma.com/file/oQsMzMMnynfPWpMEt91OpH/HDS-Product---Foundations?type=design&node-id=2130%3A2&t=6XBReWOxMRTiW1Iv-1) to your text.
 
-The Ember component inherits the styles of the text around it. Depending on the surrounding text, the link style in Figma may need to be overridden to ensure that the link and body text are the same.
+The Ember component inherits the typographic styles of the text around it (font, weight, etc), while the [color](#color) is determined by the `@color` argument.
+
+Depending on the surrounding text, the link style in Figma may need to be overridden to ensure that the link and body text are the same.
```

Comment:
> Depending on the surrounding text, the link style in Figma may need to be overridden to ensure that the link and body text are the same.

Can you clarify what you mean by this please? Do you mean the typographic style or the color?

Reply:
What I'm trying to say here is: to match the style of text around it, the default Link (`Body / 200 / Link` for example) may need to be overwritten to match the text around it because it has a slightly heavier weight. Let me try to wordsmith that a little bit, do you have any thoughts?

Reply:
I think there are two approaches to setting up the links within a paragraph in Figma: 
1. Select the text that needs to be a link and set the decoration to "Underline"
    - this method will already inherit the other typographic styles of the paragraph
2. Select the text that needs to be a link and change the font style to `Body/##/Link`
    - if the paragraph style doesn't use a `medium` weight, this will result in an incorrect weight shift and a misalignment between design and code
    - alternatively, they could update the weight manually
    - this method is more complex to do and document

It is known that a fix for this larger issue would be to offer Link styles for all weights, but this is not yet (if ever) something we plan to tackle. In the meantime, I recommend we advocate for the easier approach of updating the decoration to "Underline." So with that in mind, what about something like: 

> Since a component isn't available for this within Figma, you will need to manually adjust the style of the text. To do this, select the text needed for the link, and change the decoration to "Underline".